test_ir.py

```diff
@@ -18,5 +18,14 @@ class IRkernelTests(jkt.KernelTests):
     complete_code_samples = ['1', "print('hello, world')", "f <- function(x) {\n  x*2\n}"]
     incomplete_code_samples = ["print('hello", "f <- function(x) {\n  x*2"]
 
+class InstallspecTests(jkt.KernelTests):
+    # just a small test, it's the same kernel after all...
+    kernel_name = "testir"
+
+    language_name = "R"
+
+    code_hello_world = "print('hello, world')"
+
+
```

Comment:
> i’m sure you like working on the codebase more now that it doesn’t look like it did before.

Not especially. ;-) It's not worse, I just don't really care about style. Possibly this comes from working on lots of different projects, and on some quite old codebases - I'm used to looking at code in all sorts of styles, and the details of that don't really bother me.

The thing that annoys me is when projects have automatic style checkers and won't accept working, readable PRs until the code complies with every last niggling detail of the style guide.